core/fortran-example/tutorial/05_parallel_for/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+  add_executable(ForParallelFor Fortran_main.f90 Fortran_kernels.F90 parallel_for_example.cpp)
+  set(CMAKE_Fortran_FLAGS "-ta=tesla:cuda9.0 -Minfo=accel -Mcuda=cuda9.0")
+  set(CMAKE_CXX_FLAGS "--relocatable-device-code=true")
+  set(CMAKE_EXE_LINKER_FLAGS "-pgc++libs")
```

Comment:
Should be inside `if(COMPILER_ID = ‘PGI’)`

Reply:
Done.